docs/abc.rst

```diff
@@ -98,7 +98,7 @@ experimental APIs without issue.
             Renamed from ``_sleep()``.
 
 
-    .. coroutine:: getitem(url: str, url_vars: Dict[str, str] = {}, *, accept=sansio.accept_format()) -> Any
+    .. coroutine:: getitem(url: str, url_vars: Dict[str, str] = {}, *, accept=sansio.accept_format(), authorization=None) -> Any
```

Comment:
I just realized that I left the type info out for `accept` which led you to leave it out for `authorization` everywhere. Now I have to decide if I need to go back and add the missing type info or remove it from the docs entirely.